src/main/_docs/ide/ide-git-svn.md

```diff
@@ -184,14 +191,84 @@ Files in project explorer and editor tabs can be colored according to their Git
 * Blue: files that contain changes.
 * Yellow: files that are not staged in index.
 
-## Reviewing changes with the diff window
-With the diff window it is possible to review changes between two states of code. To view the diff, use `Git`->`Compare`->`<Select-to-what>` from main menu. If more than one file has changed the list of changed files will be opened first. To select a file to compare double-click on it or select a file then click the `Compare` button.
+## Committing changes
+To commit your changes use the git commit window. To show it click `Git` -> `Commit...` from main menu or just use `Alt + c` shortcut.
```

Comment:
Please note that it is also possible to do that from project explorer's context menu and from editor's context menu

Reply:
yes, thank you, this must be documented.